apps/web/src/pages/templates/TemplatesListNoData.tsx

```diff
@@ -154,7 +168,14 @@ export const TemplatesListNoData = ({
                     data-can-be-hidden={index === 2}
                     data-test-id="second-workflow-tile"
                     disabled={readonly || isCreating}
-                    onClick={() => onTemplateClick(template)}
+                    onClick={() => {
+                      segment.track('[Template Store] Click Create Notification Template', {
+                        templateIdentifier: template?.triggers[0]?.identifier || '',
+                        location: TemplateCreationSourceEnum.EMPTY_STATE,
+                      });
+
+                      onTemplateClick(template);
```

Comment:
we will have 2 analytic events: one from here and one triggered on the BE, is that what we want?

Reply:
That is true @LetItRock , was not sure if we want to separate those api-clients events or not. do you think we can drop this client event?

Reply:
Will merge for now we always can remove it.

Reply:
Let's merge this for now, we can always remove it.

Reply:
Let's merge this for now, we can always remove it.
@LetItRock  @scopsy